### PR TITLE
Add sections for variance errors

### PR DESCRIPTION
**Pull request by Masa-Shin**

### CODE REVIEW

File: errors/includes/TS2xxx/TS2636.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+## TS2636
+
+> error TS2636: Type 'Func<super-T>' is not assignable to type 'Func<sub-T>' as implied by variance annotation. Type 'super-T' is not assignable to type 'sub-T'.
```
Comment (bennycode) [inline errors/includes/TS2xxx/TS2636.md]:
```suggestion
> error TS2636: Type '`Func<super-T>`' is not assignable to type '`Func<sub-T>`' as implied by variance annotation. Type '`super-T`' is not assignable to type '`sub-T`'.
```

**Review by bennycode — CHANGES_REQUESTED** [review]
Hey, happy to see you again! 🙂 Let's put the variable values of an error message into code blocks so it's visible that this part of the error message is based on your custom code.

**Comment by Masa-Shin** [comment]
Thanks! Updated.

**Comment by bennycode** [comment]
Cool! It's live on https://typescript.tv/errors/#TS2636 - Thank you very much! 👍👍

**Pull request merged**